.github/workflows/cron.stale.yml

```diff
@@ -24,4 +24,5 @@ jobs:
     - name: Unassign contributor after days of inactivity
       uses: BoundfoxStudios/action-unassign-contributor-after-days-of-inactivity@v1.0.3
       with:
-        last-activity: 14
\ No newline at end of file
+        last-activity: 14
+        
```

Comment:
this looks odd, why is this file changed?

Reply:
The code check from codeRabbit said this file had an incorrect line ending.

Reply:
that doesn't mean you should add more line endings ;)